Author: Maximiliano Curia
        Andreas Tille <tille@debian.org>
Last-Update: Wed, 22 Aug 2018 16:37:59 +0200
Description: We are using Debian packaged ckeditor3
 There should be no need to fiddle around with this Makefile.am target

--- a/appsrc/ODS-Framework/Makefile.am
+++ b/appsrc/ODS-Framework/Makefile.am
@@ -56,43 +56,13 @@ EXTRA_DIST = \
 	*.xml \
 	*.xsl \
 	ckeditor/*.css \
-	ckeditor/*.html \
+	ckeditor/*.pack\
 	ckeditor/*.js \
-	ckeditor/images/*.gif \
-	ckeditor/lang/*.js \
-	ckeditor/plugins/clipboard/dialogs/*.js \
-	ckeditor/plugins/colordialog/dialogs/*.js \
-	ckeditor/plugins/dialog/*.js \
-	ckeditor/plugins/find/dialogs/*.js \
-	ckeditor/plugins/flash/dialogs/*.js \
-	ckeditor/plugins/flash/images/*.png \
-	ckeditor/plugins/image/dialogs/*.js \
-	ckeditor/plugins/link/dialogs/*.js \
-	ckeditor/plugins/link/images/*.gif \
-	ckeditor/plugins/pastetext/dialogs/*.js \
-	ckeditor/plugins/scayt/dialogs/*.css \
-	ckeditor/plugins/scayt/dialogs/*.js \
-	ckeditor/plugins/showblocks/images/*.png \
-	ckeditor/plugins/smiley/dialogs/*.js \
-	ckeditor/plugins/smiley/images/*.gif \
-	ckeditor/plugins/specialchar/dialogs/*.js \
-	ckeditor/plugins/specialchar/lang/*.js \
-	ckeditor/plugins/styles/styles/*.js \
-	ckeditor/plugins/table/dialogs/*.js \
-	ckeditor/plugins/tabletools/dialogs/*.js \
-	ckeditor/plugins/templates/dialogs/*.js \
-	ckeditor/plugins/templates/templates/*.js \
-	ckeditor/plugins/templates/templates/images/*.gif \
-	ckeditor/plugins/wsc/dialogs/*.css \
-	ckeditor/plugins/wsc/dialogs/*.html \
-	ckeditor/plugins/wsc/dialogs/*.js \
-	ckeditor/skins/kama/*.css \
-	ckeditor/skins/kama/*.js \
-	ckeditor/skins/kama/*.png \
-	ckeditor/skins/kama/images/*.gif \
-	ckeditor/skins/kama/images/*.png \
-	ckeditor/themes/default/*.js \
-	comp/*.js \
+	ckeditor/images \
+	ckeditor/lang \
+	ckeditor/plugins \
+	ckeditor/skins \
+	ckeditor/themes \
 	comp/*.xml \
 	comp/*.xsl \
 	icons/*.gif \
